src/main/java/uk/gov/hmcts/reform/sandl/snlrules/model/Session.java

```diff
@@ -48,7 +48,7 @@ public boolean isOverlapping(Session s2) {
     }
 
     @Override public String toDescription() {
-        return ("Start: " + DateTimeUtils.humanizeDate(start) + ", Case type: " + caseType)
+        return ("Start: " + DateTimeUtils.humanizeDate(start) + ", Duration " + duration + ", Case type: " + caseType)
```

Comment:
this duration could be humanized I think, otherwise, it will be formatted like this:  'PTnHnMnS' - plus it's also an issue in another place that uses duration.

Reply:
agree, preferred humanized version as it will be displayed in the UI

Reply:
as discussed the bug should be fixed with the tostring(?) implementation not in description